objects/rct2ww/scenery_wall/rct2.ww.wshogi15.json

```diff
@@ -13,15 +13,15 @@
     "images": ["$RCT2:OBJDATA/WSHOGI15.DAT[0..5]"],
     "strings": {
         "name": {
-            "en-GB": "Shōji Wall",
-            "fr-FR": "Cloison shogi",
-            "de-DE": "Shōji-Wand",
+            "en-GB": "Japanese Fence",
+            "fr-FR": "Clôture japonais",
+            "de-DE": "Japanische Hecke",
```

Comment:
That doesn't sound quite right to me. Shouldn't it be "Zaun"?

@Wuzzy2

Reply:
If that’s the case, there are a few more instances of Hecke that ought to be cleaned up. (Which should be much easier, now, with the scripts…)